### DISCUSSION TITLE
About to import: But photos will be double imported after conversion

### DISCUSSION THREAD

**Discussion by vandad51** [photoprism/photoprism#3517] [Q&A]
I'm about to do a mass import on my wife and I's iPhones but upon doing some test imports to make sure my structure will be proper, I noticed that when it converts my HEIC photos to JPEG, it keeps the HEIC. When I think about the amount of photos I'm going to be importing, that's a lot of extra HEIC photos I won't have any need for, an extra 11GB worth, 4,000 photos (doesn't include my wife's pictures yet) I won't use for any reason because I can't use HEIC for anything on my Windows computers.

I like the HEIC feature on the iPhone for better video options and of course, better efficiency with space usage. I am willing sacrifice that for the backup, as it will just render everything back into JPEG and .MOV formats that I can use with my Windows computers.

Is there a feature or setting that I'm missing that will prevent this problem? Should I delete the Original HEIC photos after import? Will that affect PhotoPrism from knowing what's all been imported?

[Link to Reddit post I made already,](https://www.reddit.com/r/photoprism/comments/14pkkyi/about_to_import_but_photos_will_be_double/) no responses at the time of posting this.

**Comment by lastzero** [comment]
Kindly note that we are on vacation for the next two weeks.

**Comment by graciousgrey** [comment]
> Should I delete the Original HEIC photos after import? Will that affect PhotoPrism from knowing what's all been imported?

Deleting original files while keeping generated sidecar files is not supported at this time. 

The JPG files that are generated by PhotoPrism are stored in the sidecar directory while original files are stored in originals. If PhotoPrism notices (e.g. during indexing) that you deleted an original file, the photo will be flagged as missing and won't show up in the UI anymore.
